internal/engine/eval/trusty/trusty.go

```diff
@@ -105,7 +105,10 @@ func (e *Evaluator) Eval(ctx context.Context, pol map[string]any, res *engif.Res
 	for _, dep := range prDependencies.Deps {
 		depscore, err := getDependencyScore(ctx, e.client, dep)
 		if err != nil {
-			logger.Error().Msgf("error fetching trusty data: %s", err)
+			logger.Error().
+				Str("dependency_name", dep.Dep.Name).
+				Str("dependency_version", dep.Dep.Version).
+				Msgf("error fetching trusty data: %s", err)
```

Comment:
**nit:** I would track the error outside of the message, making it easier to filter by it
```suggestion
			logger.Error().
				Err(err).
				Str("dependency_name", dep.Dep.Name).
				Str("dependency_version", dep.Dep.Version).
				Msgf("error fetching trusty data")
```